package/gluon-luci-node-role/files/usr/lib/lua/luci/model/cbi/admin/noderole.lua

```diff
@@ -1,23 +1,22 @@
 local f, s, o
 local site = require 'gluon.site_config'
+local i18n = require "luci.i18n"
 local uci = luci.model.uci.cursor()
 local config = 'gluon-node-info'
 
 -- where to read the configuration from
 local role = uci:get(config, uci:get_first(config, "system"), "role")
 
-f = SimpleForm("role", "Verwendungszweck")
-f.reset = false
+f = SimpleForm("role", i18n.translate("Node role"))
 f.template = "admin/expertmode"
-f.submit = "Fertig"
+f.submit
```

Comment:
The f.submit line should be removed completely. I don't even think this is correct Lua...